[docs][ao] Add missing documentation for torch.quantized_batch_norm

supriyar opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #63263
* #63258
* #63242
* __->__ #63240

Summary:
Op is exposed via torch.quantized_batch_norm to the end user without any existing documentation

Test Plan:
CI
Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D30316431](https://our.internmc.facebook.com/intern/diff/D30316431)

supriyar committed:
[docs][ao] Add missing documentation for torch.quantized_batch_norm

Summary:
Op is exposed via torch.quantized_batch_norm to the end user without any existing documentation

Test Plan:
CI
Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar committed:
Update on "[docs][ao] Add missing documentation for torch.quantized_batch_norm"


Summary:
Op is exposed via torch.quantized_batch_norm to the end user without any existing documentation

Test Plan:
CI
Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar committed:
Update on "[docs][ao] Add missing documentation for torch.quantized_batch_norm"


Summary:
Op is exposed via torch.quantized_batch_norm to the end user without any existing documentation

Test Plan:
CI
Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar commented:
@supriyar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30316431).

raghuramank100 commented on torch/_torch_docs.py:
```diff
@@ -11224,6 +11224,50 @@ def merge_dicts(*dicts):
+
+Arguments:
+    input (Tensor): quantized tensor
+    weight (Tensor): tensor that corresponds to the gamma, size C
```
Can we also specify that weight , bias and other tensors are in floating point?

supriyar committed:
Update on "[docs][ao] Add missing documentation for torch.quantized_batch_norm"


Summary:
Op is exposed via torch.quantized_batch_norm to the end user without any existing documentation

Test Plan:
CI
Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D30316431](https://our.internmc.facebook.com/intern/diff/D30316431)

[ghstack-poisoned]

supriyar commented:
@supriyar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30316431).
